tests/app/main/views/test_services.py

```diff
@@ -560,17 +555,15 @@ def test_service_update_with_one_service_feature(self, data_api_client):
                 'serviceBenefits': 'foo',
             }
         )
-        assert data_api_client.update_service.call_args_list == [(
-            (
+        assert data_api_client.update_service.call_args_list == [
+            mock.call(
```

Comment:
Definitely more readable with `mock.call`!

Reply:
...and also I couldn't for the life of me figure out how to assert the new named parameter the other way 😛